### ISSUE TITLE
Got an error from the Unified sampler: IndexError: index 1 is out of bounds for dimension 0 with size 1

### ISSUE THREAD

**Issue by n0valis** [flowstateeng/FlowState-Creator-Nodes#2]
So this is the traceback
` File "G:\AITools\ComfyUI\ComfyUI\custom_nodes\FlowState-Nodes\FlowStateUnifiedSampler.py", line 585, in execute
    batch_img, batch_latent, batch_params = self.sample(
                                            ^^^^^^^^^^^^`

`  File "G:\AITools\ComfyUI\ComfyUI\custom_nodes\FlowState-Nodes\FlowStateUnifiedSampler.py", line 448, in sample
    latent_out = self.sample_flux(
                 ^^^^^^^^^^^^^^^^^`

`  File "G:\AITools\ComfyUI\ComfyUI\custom_nodes\FlowState-Nodes\FlowStateUnifiedSampler.py", line 143, in sample_flux
    conditioning = conditioning_set_values(positive_conditioning, {'guidance': guidance})
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^`

 ` File "G:\AITools\ComfyUI\ComfyUI\node_helpers.py", line 10, in conditioning_set_values
    n = [t[0], t[1].copy()]`

For positive conditioning I am using CLIPTextEncodeFlux from core which includes guidance. Is that a problem?

any idea?

**Comment by flowstatelabs** [comment]
yeah probably will be a problem. the sampler handles guidance internally for you. just go straight from clip to text encode to the sampler

**Comment by n0valis** [comment]
The sampler does not differentiate split positive prompts clip_l - t5xxl
